### DISCUSSION TITLE
0.9.5?

### DISCUSSION THREAD

**Discussion by DatEmulationGuy** [rafaelvcaetano/melonDS-android#990] [Q&A]
This great update got released a hour ago, will we get it's changes? Very interested about the local multiplayer and dsi camera

**Comment by rafaelvcaetano** [comment]
Of course! I was kind of waiting for it before releasing a new update. But don't expect the new local multiplayer to work. It improves stability when running 2 instances on the same machine, which doesn't actually make sense in Android. It's possible to run 2 instances by using cloning apps, but I still have to take a look at the new multiplayer implementation to see if there's anything usable that can be done in Android.
Regarding the DSi camera, that shouldn't be a problem (hopefully).

**Reply by DatEmulationGuy** [reply]
Is possible to do local multiplayer like my boy and pizza boy emulators?
